### PR TITLE
CASMINST-5443: Authenticate to Artifactory

### PR DESCRIPTION
**Pull request by jsollom-hpe**
## Summary and Scope

Authenticate to Artifactory

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMINST-5443
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

* No.
### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? No
- Were continuous integration tests run? If not, why? No
- Was upgrade tested? If not, why? No
- Was downgrade tested? If not, why? No
- Were new tests (or test issues/Jiras) created for this change? No

## Risks and Mitigations
Low.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: manifests/sysmgmt.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@ spec:
     namespace: services
   - name: cray-cfs-operator
     source: csm-algol60
-    version: 1.17.0-beta.4
+    version: 1.16.2
```
Comment (mtupitsyn) [inline manifests/sysmgmt.yaml:81]: Is that expected that `cray-cfs-operator` and `cfs-hwsync-agent` versions are decreased?

Comment (jsollom-hpe) [reply manifests/sysmgmt.yaml:81]: Yes, one of our developers was incorrectly tagging the development branch with beta tags. This is software that was not released and the versions were not correct, so the decrease in version is okay.

File: manifests/sysmgmt.yaml [hunk 2]
```diff
@@ -90,11 +90,11 @@ spec:
     namespace: services
   - name: cfs-trust
     source: csm-algol60
-    version: 1.6.1
+    version: 1.6.2
     namespace: services
   - name: cfs-hwsync-agent
     source: csm-algol60
-    version: 1.9.0-beta.1
+    version: 1.8.1
```
Comment (jsollom-hpe) [inline manifests/sysmgmt.yaml:97]: I need to move the change into the release branch, which is tagged as beta, and increment it. I will do so and resolve this comment once I have done that.

File: manifests/sysmgmt.yaml (hunk 1 above)
Comment (jsollom-hpe) [reply manifests/sysmgmt.yaml]: Fixed.

File: manifests/sysmgmt.yaml (hunk 2 above)
Comment (jsollom-hpe) [reply manifests/sysmgmt.yaml]: Fixed.

**Comment by jsollom-hpe** [comment]
This pull request has been superseded by https://github.com/Cray-HPE/csm/pull/1598/files. Therefore, I am closing it.

**Pull request closed without merging**